merlin-server/sql/merlin/tables/constraint.sql

```diff
@@ -82,3 +82,20 @@ create trigger set_timestamp
 before update on "constraint"
 for each row
 execute function constraint_set_updated_at();
+
+create or replace function constraint_check_constraint_run()
```

Comment:
With the SQL code, we use declarative `create`s outside of migrating function definitions, since we *know* that the function does not exist.
```suggestion
create function constraint_check_constraint_run()
```

Reply:
👍